README.md

```diff
@@ -1,2 +1,25 @@
 # Seine-HelloWorld
 The testing repo for Seine Bot
+
+
+# Assignment 1 (5/19/24) Test Event Types for Seine AI
+
+1. Fork this repository. Create/Edit a file of your choice.
+2. Create a PR with your commmited changes
+3. Test out the following event types:
+ - add PR Request review
+ - specific line comment review 
+
+Demos below have more details:
```

Comment:
+
+## Demo 1: Add PR Request Review
+
+1. Fork this repository
+2. Create a new branch
+3. Add a new file
+4. Commit the changes
+5. Create a PR
+6. Request a review from yourself
+7. Check the PR page for the review request
+
+## Demo 2: Specific Line Comment Review
+
+1. Fork this repository
+2. Create a new branch
+3. Add a new file
+4. Commit the changes
+5. Create a PR
+6. Comment on a specific line in the PR
+7. Check the PR page for the comment
+
+## Demo 3: Testing PR Request Review and Specific Line Comment Review
+
+1. Fork this repository
+2. Create a new branch
+3. Add a new file
+4. Commit the changes
+5. Create a PR
+6. Request a review from yourself
+7. Comment on a specific line in the PR
+8. Check the PR page for the review request and comment


---

Thanks for the review!

I've added the assignment details as requested.

For the event types, I've added demos for Add PR Request Review, Specific Line Comment Review, and Testing PR Request Review and Specific Line Comment Review.

Each demo includes step-by-step instructions for testing the event type.

Please let me know if there's anything else I can do to improve this PR.